stdlib/src/sys/info.mojo

```diff
@@ -446,6 +446,32 @@ fn is_big_endian[
     ]
 
 
+@always_inline("nodebug")
+fn is_32bit[target: __mlir_type.`!kgen.target` = _current_target()]() -> Bool:
+    """Returns True if the maximum integral value is 32 bit.
+
+    Parameters:
+        target: The target architecture.
+
+    Returns:
+        True if the maximum integral value is 32 bit, False otherwise.
+    """
+    return sizeof[DType.index, target]() == sizeof[DType.int32, target]()
```

Comment:
what do you think about this?

Reply:
LGTM.  Seems like a much more natural place to put this function. 👍 

Thanks!